### PR TITLE
CIV-3764 cant use this service 18 or over

### PR DESCRIPTION
**Pull request by hmcts-version1-domini**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-3764

### Change description ###

Develop the 'You can't use this service' - You need to be 18 or over to use this service.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/common/form/models/eligibility/NotEligibleReason.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export enum NotEligibleReason {
   CLAIM_VALUE_OVER_25000 = 'claim-value-over-25000',
   UNDER_18 = 'under-18',
   UNDER_18_DEFENDANT = 'under-18-defendant',
+  UNDER_18_CLAIMANT = 'under-18',
```
Comment (portizdev) [inline src/main/common/form/models/eligibility/NotEligibleReason.ts:7]: I think you have duplicated the constant, in line 5 you have 'under-18' I think it is the same

File: src/main/common/utils/notEligibleReasonConvertor.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const convertToNotEligibleReason = (reason: string): NotEligibleReason =>
     case 'claim-value-over-25000':
       return NotEligibleReason.CLAIM_VALUE_OVER_25000;
     case 'under-18':
-      return NotEligibleReason.UNDER_18;
+      return NotEligibleReason.UNDER_18_CLAIMANT;
```
Comment (portizdev) [inline src/main/common/utils/notEligibleReasonConvertor.ts:12]: Why this change?

**Review by portizdev — CHANGES_REQUESTED** [review]

File: src/main/common/form/models/eligibility/NotEligibleReason.ts (hunk 1 above)
Comment (hmcts-version1-domini) [reply src/main/common/form/models/eligibility/NotEligibleReason.ts:6]: Ok. Removed the generic one as we have talked about.

File: src/main/common/utils/notEligibleReasonConvertor.ts (hunk 2 above)
Comment (hmcts-version1-domini) [reply src/main/common/utils/notEligibleReasonConvertor.ts:12]: because I wanted the constants to have the name with the same structure as the UNDER 18 DEFENDANT

**Review by portizdev — APPROVED** [review]
